editor/editor_tests/CMakeLists.txt

```diff
@@ -18,7 +21,21 @@ omim_add_test(${PROJECT_NAME} ${SRC})
 omim_link_libraries(
   ${PROJECT_NAME}
   editor
+  editor_tests_support
   platform_tests_support
+  generator_tests_support
+  generator
+  indexer_tests_support
```

Comment:
Move this to other _tests_supports, please.

Reply:
done